.github/workflows/e2e-test.yml

```diff
@@ -324,6 +324,9 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.17.3'
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
```

Comment:
@andyi2it : Curious to know how did you figure out from the [logs](https://github.com/kserve/kserve/actions/runs/4315581912/jobs/7531712384) that it was python version issue.

Would you mind sharing how you figured this out?

Reply:
@rachitchauhan43 
Its not from the e2e logs alone.
I ran it locally, and compared the differences. My local had an earlier version of python and it was working.
So, I tried pinning the version. 
However, the most accurate detail is that, the portforward library was the one that was breaking the e2e.
The newer versions of python uses the newer portforward library by default and hence the error.

Given that the root cause is identified, we can remove pinning python to 3.9. 
However, I think we should pin python to the max version we support. Will create a separate issue for that.

Reply:
@andyi2it : This is really awesome work. Thanks for this fix 👍